installation/security.md

```diff
@@ -209,6 +215,15 @@ Open the configuration file and **add** the following lines underneath the proxy
         auth_basic_user_file                  /etc/nginx/.htpasswd;
 ```
 
+##### Add authorization and cookie directives in NGINX Configuration
+
+This is an important new requirment in openHAB 3.0 and later versions.  This is not required prior to openHAB 3.0. In the `location /` block, you must add the following two directives underneath the `add_header` and `proxy_set_header` items respectively:
```

Comment:
Please use one line per sentence here.
The website will take care of a correct formatting.